agent/wptdriver/wpt_test.h

```diff
@@ -225,6 +225,11 @@ class WptTest {
   void      BuildScript();
   CAtlList<ScriptCommand> _script_commands;
 
+  //Timeout information
+  DWORD    _timeout_value_in_seconds;
+  bool     _enable_per_test_timeout;

```

Comment:
I don't think you need _enable_per_test_timeout in the WptTest as well. Having it in WptSettings should suffice as an instance of it is passed to WebBrowser() and that's the only place you need _enable_per_test_timeout.